src/main/java/marustuff/textometer/controller/RequestController.java

```diff
@@ -21,55 +21,65 @@
     @NonNull
     @Autowired
     private final MeteringRepository repository;
+    public final String submitWordReturn="redirect:/request/get/";
+    public final String submitVSReturn="redirect:/request/getvs/";
+    public final String testMeteringReturn="debugMeteringRepository";
+    public final String serveMeteringModelAttributeName="currentWord";
+    public final String serveMeteringReturn="request";
+    public final String serveMeteringReturnIfPolling="redirect:/poll/";
+    public final String serveVsMeteringModelAttributeName="currentVs";
+    public final String serveVsMeteringReturnIfFound="vs";
+    public final String serveVsMeteringReturnIfNotFound="redirect:/poll/";
 
     @PostMapping("/post")
     public String submitWord(@ModelAttribute Metering metering){
-        return "redirect:/request/get/"+metering.getWord();
+        return submitWordReturn+metering.getWord();
     }
 
     @PostMapping("/postvs")
     public String submitVS(@ModelAttribute Vs vs){
-        return "redirect:/request/getvs/"+vs.getWord1()+"/"+vs.getWord2();
+        return submitVSReturn+vs.getWord1()+"/"+vs.getWord2();
     }
     @GetMapping("/debug/metering")
     public String TestMetering(Model model){
         model.addAttribute("currentWords",repository.findAll());
-        return "debugMeteringRepository";
+        return testMeteringReturn;
     }
 
     @GetMapping("get/{word}")
-    public String ServeMetering(@PathVariable String word, Model model) {
+    public String serveMetering(@PathVariable String word, Model model) {
+        //findById zwraca Optional, czyli wartość tutaj może nie zostać zwrócona. Należy najpierw sprawdzić jej obecność poprzez isPresent() a dopiero potem ewentualnie ją pobierać get()
         if(repository.existsById(word)&&isItFresh(repository.findById(word).get())){
-            model.addAttribute("currentWord",repository.findById(word).get());
+            //findById zwraca Optional, czyli wartość tutaj może nie zostać zwrócona. Należy najpierw sprawdzić jej obecność poprzez isPresent() a dopiero potem ewentualnie ją pobierać get()
+            model.addAttribute(serveMeteringModelAttributeName,repository.findById(word).get());
```

Comment:
optional wymaga sprawdzenia czy zawiera w sobie obiekt

Reply:
nie będziesz musiał tego pobierać ponownie jeśli już będziesz miał to przypisane do zmiennej